src/assets/css/global/all.css

```diff
@@ -28,6 +28,7 @@ input {
   @apply bg-white dark:bg-gray-900;
   @apply text-gray-900 dark:text-gray-50;
   min-height: calc(100vh + 1px);
+  height: calc(100vh + 1px);
```

Comment:
Need to add height so we can actually infer `height: 100%` for child components. In this PR, you'll see it for `FocusedLayout`